source/heatfluxClass.py

```diff
@@ -554,6 +555,34 @@ def multiExp_profile_fluxspace(self, PFC, R, Bp, psiN, mode):
         q[nan_locations] = 0.0
         q[inf_locations] = 0.0
         return q
+    
+    def from_rzq_profile(self, rzq_data, PFC):
+        """"
+        q from any R(m), Z(m), q(W/m2) data
+        q is mapped from the R,Z grid to OMP
```

Comment:
i think a bit more elaborate description would help future users understand what you are doing here

Reply:
I edited this comment to give more explanation